src/main/java/Duke/Ui.java

```diff
@@ -0,0 +1,183 @@
+package Duke;
+
+import Duke.Tasks.Task;
+import java.util.Scanner;
+
+/**
+ * Represent Ui class to interact with user.
+ */
+public class Ui {
+    protected Scanner sc;
+
+    /**
+     * Constructor for the Ui class.
+     */
+    public Ui() {
+        this.sc = new Scanner(System.in);
+    }
+
+    /**
+     * Gets input entered by user.
+     *
+     * @return String Input entered by user.
+     */
+    public String readCommand() {
+        return this.sc.nextLine();
+    }
+
+    /**
+     * Prints greeting message to the user.
+     */
+    public void sayHello() {
+        String greetMessage = String.format(
+                "____________________________________________________________\n" +
+                        " Hello! I'm Corgi!\n" +
+                        " What can I do for you?\n" +
+                        "____________________________________________________________\n");
+        System.out.println(greetMessage);
+    }
+
+    /**
+     * Prints goodbye message to the user.
+     */
+    public void sayGoodbye() {
+        String exitMessage = "____________________________________________________________\n" +
+                " Bye. Hope to see you again soon!\n" +
+                "____________________________________________________________\n";
+        System.out.println(exitMessage);
+    }
```

Comment:
I like the distinction in names between methods that are called once vs methods that can be called multiple times.